### PR TITLE
Added AdminGraphqlApiId to Image struct

### PR DESCRIPTION
**Pull request by vinhluan**

### CODE REVIEW

File: image.go [hunk 1]
```diff
@@ -36,6 +36,7 @@ type Image struct {
 	Attachment string     `json:"attachment,omitempty"`
 	Filename   string     `json:"filename,omitempty"`
 	VariantIds []int64    `json:"variant_ids,omitempty"`
+	AdminGraphqlApiId     `json:"admin_graphql_api_id,omitempty"`
```
Comment (oliver006) [inline image.go:39]: Ah yes, this should be `AdminGraphqlApiId string  ....`

**Review by oliver006 — COMMENTED** [review]
Thanks for the PR - you're missing the "string" type and then this should be good to go.

File: image.go (hunk 1 above)
Comment (vinhluan) [reply image.go]: Yeah my bad, I was editing the file directly on GitHub 🥲

**Review by oliver006 — APPROVED** [review]
👍

**Pull request merged**